moon/powerups/server/regen.moon

```diff
@@ -52,6 +52,9 @@ class RegenPowerup extends BasePowerup
         -- Make sure they don't have more 100 HP
         plyHealth = @owner\Health!
         if plyHealth > 100
```

Comment:
```suggestion
        if @owner\Health! > 100
```
Reason being because this value is no longer valid after you `@owner\SetHealth`

You know what would be cool? If we noted the owner's health at the time of getting the powerup, and the base powerup had a `restoreOwnerHealth` method

Reply:
My thinking was this future proofs it if we ever want to add extra notifs in there, "You've had your health reset to `plyHealth`!"